src/Components/v2/CountrySelect.tsx

```diff
@@ -205,7 +216,7 @@ const COUNTRY_SELECT_OPTIONS = [
   { text: "Sao Tome and Principe", value: "ST" },
   { text: "Saudi Arabia", value: "SA" },
   { text: "Senegal", value: "SN" },
-  { text: "Serbia and Montenegro", value: "CS" },
```

Comment:
And I believe proper way to refer to this country is 2 countries: https://en.wikipedia.org/wiki/ISO_3166-2:CS